### DISCUSSION TITLE
Please tell me your favorite sport!

### DISCUSSION THREAD

**Discussion by i14kawanaka** [NAIST-SE/AttractingContributors#211] [Show and tell]
https://github.com/i14kawanaka/AttractingContributors

**Comment by naistclass** [comment]
I have sent a Pull Request. Please check.
And will you help me?

https://github.com/naistclass/AttractingContributors-FavoriteFood

**Comment by TakumiHiraide** [comment]
I added !!. Please help me!!
https://github.com/TakumiHiraide/Where-do-you-want-to-go-

**Comment by ellielo** [comment]
My favorite sports on TV is the Ultimate Fighting Championship, I even installed a special [app](https://www.firesticktricks.com/watch-ufc-on-firestick.html) on my firestck to be able to stream all the important games.I also follow EU football championship at the moment, I really think Germany deserves to be a winner this year!

**Comment by CodyBra** [comment]
Sport is my life! I really love all outdoor sports, when you ride something or swim. I just love kayaks and canoes for the fact that you can go almost anywhere and find places that you could not find just on foot. I love rollerblading and skate because I can drive around the city with friends or just hang out at the skatepark. All this brings not only pleasure, but also stimulates the production of the hormone of happiness due to physical exertion, which makes sports not only useful, but simply necessary in the life of every person!

**Comment by olehvgrn** [comment]
Tennis is my favorite!
